les/server.go

```diff
@@ -49,9 +49,9 @@ type LesServer struct {
 	servingQueue *servingQueue
 	clientPool   *clientPool
 
-	freeCapacity uint64 // The minimal client capacity used for free client.
-	threadsIdle  int    // Request serving threads count when system is idle.
-	threadsBusy  int    // Request serving threads count when system is busy(block insertion).
+	minCapacity, maxCapacity, freeCapacity uint64
```

Comment:
Is it worthwhile to maintain `minCapacity ` and `maxCapacity `? Basically these two fields are the limit of capacity and we can re-calculate them via freeCapacity

Reply:
`maxCapacity` does not follow from `freeCapacity` directly. `minCapacity` always equals `freeCapacity` in the current implementation but they are two different concepts and maybe at one point we would like to allow setting `freeCapacity` higher than the absolute minimum.